tests/conftest.py

```diff
@@ -265,7 +265,11 @@ def mapdl(request, tmpdir_factory):
         port = MAPDL_DEFAULT_PORT
 
     mapdl = launch_mapdl(
-        EXEC_FILE, override=True, run_location=run_path, cleanup_on_exit=cleanup
+        EXEC_FILE,
+        override=True,
+        run_location=run_path,
+        cleanup_on_exit=cleanup,
+        additional_switches="-smp",
```

Comment:
```suggestion
```

We cannot test only on the ``smp`` version. Please check the command I mentioned above to turn off the distributed computing

Anyways, the CICD so far runs on SMP only.